SR2020/2020-03-05-coronavirus.md

```diff
@@ -0,0 +1,22 @@
+---
+to: sr2020-teams
+subject: Update on Coronavirus and SR
+---
+
+# Update on SR and the COVID-19 Coronavirus
+
+Hello everyone, 
+
+I'm sure you've heard about the COVID-19 coronavirus, and about many large events being cancelled due to concerns of spreading the virus.
+
+We're keeping a keen eye on the situation, and will be following the advice of the [UK Department of Health and Social Care][uk-coronavirus-page]. Considering this is an event for minors, we will be taking decisions with an abundance of caution.
+
+The competition is still a few weeks away and our plans will adapt to suit the situation at the time, however we are currently intending to heighten hygiene levels during the competition. For example, by frequently disinfecting any touch points (i.e. door handles & surfaces) during the competition, and we will try to source additional hand sanitizer for use during the event (however there is currently a global shortage). We'll also be briefing all volunteers and attendees about best hygiene practises.
```

Comment:
There's some grammar oddities in this sentence. 

I also don't think it's necessary to mention increasing hygiene, and simply mentioning that we'll make sure people follow best practice is probably enough.

```suggestion
The competition is still several weeks away and our plans will adapt to suit the situation as things develop. We'll also be briefing all volunteers and attendees about best hygiene practices, and ensuring they're followed throughout the competition.
```

Reply:
This is my main concern with the original